NEWS.md

```diff
@@ -83,7 +83,7 @@
 * Add option to show the R Markdown render command used when knitting (#1658)
 * Add option to show hidden files in the Files pane (#1769)
 * RStudio uses the fontconfig database to list monospace fonts when available (Linux only; #3897)
-* Upgrade embedded Pandoc to 2.5 (#1807)
```

Comment:
I think we should just say "Pandoc 2" without qualifiers here so we don't have to keep updating the comment! :-)